lib/src/wip/autocomplete/popup.dart

```diff
@@ -13,23 +13,26 @@ class Popup extends StatefulWidget {
   final Color? backgroundColor;
   final PopupController controller;
   final FocusNode parentFocusNode;
+  final Offset? offset;
 
-  Popup({
-    Key? key,
+  const Popup({
+    super.key,
     required this.normalOffset,
     required this.flippedOffset,
     required this.controller,
     required this.editingWindowSize,
     required this.style,
     required this.parentFocusNode,
+    required this.offset,
```

Comment:
```suggestion
    required this.editorOffset,
```
Add doc comments to all 3 offsets.

Reply:
done